views/poll.pug

```diff
@@ -27,4 +27,18 @@ block content
             if !choice.voted
               button(class='ui button add-vote' name=choice.name) Add vote
             else 
-              button(class='ui positive button remove-vote' name=choice.name) Remove vote
\ No newline at end of file
+              button(class='ui positive button remove-vote' name=choice.name) Remove vote
+      if !settings.addChoices
```

Comment:
shouldn't this logically be `settings.addChoices`?

Reply:
yeah -- when you change the name but not the function 😅